helm/cluster/templates/cluster-api/cluster.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  annotations:
+    {{- with .Values.metadata.description }}
+    cluster.giantswarm.io/description: "{{ . }}"
+    {{- end }}
+  labels:
+    {{- include "labels.common" $ | nindent 4 }}
+    cluster-apps-operator.giantswarm.io/watching: ""
+    {{- if .Values.metadata.labels }}
+    {{- range $key, $val := .Values.metadata.labels }}
+    {{ $key }}: {{ $val | quote }}
+    {{- end }}
+    {{- end }}
+  name: {{ include "resource.default.name" $ }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  clusterNetwork:
+    services:
```

Comment:
Do we wanna sort it similar to the CR's?
That would mean starting with pods and then services in `clusterNetwork`

Reply:
Sure :+1:

Reply:
Agree